packages/cashier/src/Stores/Cashier/cashier-store.js

```diff
@@ -462,8 +462,8 @@ export default class CashierStore extends BaseStore {
 
                 if (this.root_store.client.is_logged_in) {
                     await this.getAdvertizerError();
-                    this.checkP2pStatus();
                     this.account_prompt_dialog.resetLastLocation();
+                    if (!this.root_store.client.switched) this.checkP2pStatus();
```

Comment:
Why was this check added?

Reply:
@nijil-binary because `switched` gets empty after the account is switched so the function call happens 2 times, and in the first call `is_p2p_visible` is not filled so the correct scenario doesn't happen.